conseil-api/src/main/resources/metadata/tezos.delphinet.conf

```diff
@@ -0,0 +1,1396 @@
+            min-match-length: 4,
+            max-result-size: 100
+          }
+          value-map: {
+            "tz1gk3TDbU7cJuiBRMhwQXVvgDnjsxuWhcEA": "Airfoil"
+            "tz1MJx9vhaNRSimcuXPK2rW4fLccQnDAnVKJ": "AirGap"
+            "tz3e75hU4EhDU3ukyJueh5v6UvEHzGwkg3yC": "At James"
+            "tz1ei4WtWEMEJekSv8qDnu9PExG6Q8HgRGr3": "Bake Tz"
+            "tz1NortRftucvAkD1J58L32EhSVrQEWJCEnB": "Bake'n'Rolls"
+            "tz1S8MNvuFEUsWgjHvi3AxibRBf388NhT1q2": "Binance"
+            "tz1KtGwriE7VuLwT3LwuvU9Nv4wAxP7XZ57d": "Bitfinex"
+            "tz1SYq214SCBy9naR6cvycQsYcUGpBqQAE8d": "Coinone"
+            "tz1eEnQhbwf6trb8Q8mPb2RaPkNk2rN7BKi8": "Cryptium Labs"
+            "tz1MXFrtZoaXckE41bjUCSjAjAap3AFDSr3N": "Everstake"
+            "tz1TzaNn7wSQSP5gYPXCnNzBCpyMiidCq1PX": "Flippin' tacos"
+            "tz3RDC3Jdn4j15J7bBHZd29EUee9gVB1CxD9": "Foundation Baker 1"
+            "tz3bvNMQ95vfAYtG8193ymshqjSvmxiCUuR5": "Foundation Baker 2"
+            "tz3RB4aoyjov4KEVRbuhvQ1CKJgBJMWhaeB8": "Foundation Baker 3"
+            "tz3bTdwZinP8U1JmSweNzVKhmwafqWmFWRfk": "Foundation Baker 4"
+            "tz3NExpXn9aPNZPorRE4SdjJ2RGrfbJgMAaV": "Foundation Baker 5"
+            "tz3UoffC7FG7zfpmvmjUmUeAaHvzdcUvAj6r": "Foundation Baker 6"
+            "tz3WMqdzXqRWXwyvj5Hp2H7QEepaUuS7vd9K": "Foundation Baker 7"
+            "tz3VEZ4k6a4Wx42iyev6i2aVAptTRLEAivNN": "Foundation Baker 8"
+            "tz1NpWrAyDL9k2Lmnyxcgr9xuJakbBxdq7FB": "Gate.io"
+            "tz1WCd2jm4uSt4vntk4vSuUWoZQGhLcDuR9q": "Happy Tezos"
+            "tz1gfArv665EUkSg2ojMBzcbfwuPxAvqPvjo": "Kraken"
+            "tz1X1fpAZtwQk94QXUgZwfgsvkQgyc2KHp9d": "ownBLOCK (Legacy)"
+            "tz1hTFcQk2KJRPzZyHkCwbj7E1zY1xBkiHsk": "ownBLOCK"
+            "tz1P2Po7YM526ughEsRbY4oR9zaUPDZjxFrb": "P2P Validator"
+            "tz1KtvGSYU5hdKD288a1koTBURWYuADJGrLE": "Ø Crypto Pool"
+            "tz1Yju7jmmsaUiG9qQLoYv35v5pHgnWoLWbt": "Polychain Labs"
+            "tz2TSvNTh2epDMhZHrw73nV9piBX7kLZ9K9m": "Polychain Labs 2"
+            "tz1Y42aKCk53vMbaJNpf1hBg1rznGdBxHJ5C": "Tezos Berlin"
+            "tz1Vyuu4EJ5Nym4JcrfRLnp3hpaq1DSEp1Ke": "POS Bakerz"
+            "tz1b7YSEeNRqgmjuX4d4aiai2sQTF4A7WBZf": "Tezos Japan (Legacy)"
+            "tz1ijdEfmUoQXJfSiKCLRioeCfSrKP3sVsff": "Tezos Japan"
+            "tz1PPUo28B8BroqmVCMMNDudG4ShA2bzicrU": "Tezos Korea"
+            "tz1b3SaPHFSw51r92ARcV5mGyYbSSsdFd5Gz": "Tezos MX"
+            "tz1Kf25fX1VdmYGSEzwFy1wNmkbSEZ2V83sY": "Tezos Seoul"
+            "tz1RAzdkp1x5ZDqms4ZUrSdfJuUYGH9JTPK2": "Tezos Spanish (Legacy)"
+            "tz1hAYfexyzPGG6RhZZMpDvAHifubsbb6kgn": "Tezos Suisse"
+            "tz1VYQpZvjVhv1CdcENuCNWJQXu1TWBJ8KTD": "Tezos Tokyo"
+            "tz1R6Ej25VSerE3MkSoEEeBjKHCDTFbpKuSX": "TezosSEAsia"
+            "tz1Lh9jeLSWDHYy8AshvG2dpNQseDaHg7cms": "TezosSpanish"
+            "tz1PesW5khQNhy4revu2ETvMtWPtuVyH2XkZ": "Tz Dutch"
+            "tz1Pwgj6j55akKCyvTwwr9X4np1RskSXpQY4": "Validators.com"
```

Comment:
Question: are these kept as-is on delphi too? Or could they change?

Reply:
Delphi is a testnet so they will be different. Does that mean we should remove them from the PR?

Reply:
maybe @anonymoussprocket or @vishakh knows better what to do

Reply:
We should remove them from the PR.

Reply:
They have been removed